README.md

```diff
@@ -1208,6 +1209,46 @@ API examples:
 
 [![](https://img.youtube.com/vi/sHj_3h_sX7M/mqdefault.jpg)](https://www.youtube.com/watch?v=sHj_3h_sX7M)
 
+### Module: ONVIF
+
+This module provides an **ONVIF server** that allows go2rtc to act as an ONVIF-compatible device, making it easier to integrate cameras with ONVIF-supported software like Dahua NVRs or Home Assistant.
+
+With ONVIF support, go2rtc can:
+- Expose configured streams as ONVIF profiles.
+- Provide additional ONVIF functionalities like `GetOSDs` to show camera name in Dahua NVR.
+- Maintain a **consistent camera order** to prevent issues with NVRs that rely on `GetProfilesResponse` for identification.
+
+**Example Configuration**
+
+```yaml
+onvif:
+  - name: Camera 1
+    main_stream: camera1
+    sub_stream: camera1_lq
```

Comment:
I think we should think on how this config will look like in the future. How about:

```yaml
onvif:
  - name: Camera 1
    streams:
      - camera1
      - camera1_lq
```

The first one would always be mainstream, while the second would be substream if present.

By the way some cameras have additional streams, how are they usually exposed in this case? Maybe we should support it too.

This approach is better because it doesn't require to document `third_stream`, `fourth_stream`, options. Also, it's less things to type.

As per resolution, maybe this?

```yaml
onvif:
  - name: Camera 1
    streams:
      - camera1#res=1920x1080
      - camera1_lq#res=1270x720
```

Very go2rtc-style. :)

Reply:
Another thing is, you are adding `onvif` at the top as an array of cameras. But what if in the future we may want to other introduce `onvif` global configurations?

To protect against this, I think you should add a nested option within the top `onvif`:

```yaml
onvif:
  profiles:
    - name: Camera 1
      # ...
```